src/ui/components/PromotedBadge/styles.scss

```diff
@@ -113,7 +113,7 @@
     border-color: $green-80;
     color: $green-80;
 
-    .IconPromotedBadge-shellPath--line {
+    .IconPromotedBadge-shellPath--verified {
```

Comment:
what's `-shellPath-` and why is it changing from line to verified?

Reply:
This is what controls the color of the icon that is generated. It's changing from `line` to `verified` because it should have been `verified` all along. It's a bug.